database/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -24,7 +24,7 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
                                      (implicit ec: ExecutionContext):
   Future[Seq[WorkflowStoreEntry]] = {
 
-    val action = for {
+    val action: DBIO[Seq[WorkflowStoreEntry]] = for {
```

Comment:
I think this change should be rolled back, it was meant as just a temporary thing when we were fighting through the `.result` problem yesterday.